### DISCUSSION TITLE
Switch off diagnostics via rose-edit?

### DISCUSSION THREAD

**Discussion by GiorgiaLine** [MetOffice/simulation-systems#94] [UM]
Could anyone tell me how best to swich off diagnistics/stash output via the rose-edit GUI?

**Comment by mcdalvi** [comment]
Hi, 
If the diagnostics/ stash requests are arranged in packages, you can 'Disable all packages' in the Stash requests panel. 
This will still create empty output files so these will be have to be deactivated in  the 'Model Output Streams' panel (and possibly deactivate all 'Usage' profiles as well).

![image](https://github.com/MetOffice/simulation-systems/assets/58813701/aed3f4be-f870-4b94-a590-cc42b2becf40)
